live-examples/css-examples/text-decoration/text-emphasis.html

```diff
@@ -7,7 +7,7 @@
     </div>
 
     <div class="example-choice">
-        <pre><code class="language-css">text-emphasis: filled red;</code></pre>
+        <pre><code class="language-css">text-emphasis: filled #ffb703;</code></pre>
```

Comment:
Could we please keep the red color in here? Contrast for it is fine and it's good to have a different color in the 2 choice boxes.